q2_feature_table/tests/test_normalize.py

```diff
@@ -27,6 +27,14 @@ def test_rarefy(self):
         self.assertEqual(set(a.ids(axis='observation')), set(['O1', 'O2']))
         npt.assert_array_equal(a.sum(axis='sample'), np.array([2., 2.]))
 
+    def test_rarefy_seed_1(self):
+        t = Table(np.array([[0, 1, 3], [1, 1, 2]]),
+                  ['O1', 'O2'],
+                  ['S1', 'S2', 'S3'])
+        a = rarefy(t, 2, random_seed=1)
+        self.assertEqual(a.data('S2', axis='sample').tolist(), [1, 1])
+        self.assertEqual(a.data('S3', axis='sample').tolist(), [1, 1])
+
```

Comment:
I would implement a similar test here, b/c even if a randomized seed is used you could get unlucky here and have the tables match anyway. I think this might be the case in your current implementation (see my comment above about passing a random seed through to `Table.subsample`). Alternatively, you could keep your existing `self.assertEquals` statements in this case, but I would put these in a loop over `n_iterations` to make sure you're not getting unlucky. 

```suggestion
    def test_rarefy_seed_is_not_randomized(self):
        t = Table(np.array([[0, 1, 3], [1, 1, 2]]),
                  ['O1', 'O2'],
                  ['S1', 'S2', 'S3'])
        a = rarefy(t, 2, random_seed=1)
        a_eq_b = []
        n_iterations = 100
        for i in range(n_iterations):
            b = rarefy(t, 2, random_seed=1)
            a_eq_b.append(a == b)
        self.assertFalse(False in a_eq_b,
                                  f"After {n_iterations} iterations, at least one feature "
                                  "table differed from the others. It therefore seems that a "
                                  "randomized seed is being used.")